From cfc795be2ed3a62e0ae26dd5692fe5c4e66b5776 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
Date: Wed, 13 Feb 2019 21:43:04 +0900
Subject: [PATCH 1/2] docs: fix warning for invalid code examples

For compiling docs with some version of sphinx-build, for example 1.8.4
on Ubuntu 18.04, code syntax in code-block should be valid. This example
is invalid and warned because it includes `....` even if it is just an
example.

    ..  code-block:: json

        [
            {
                "type": "primary"
            },
            ....
        ]

This patch is to correct syntax of code examples.

Signed-off-by: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
---
 docs/guides/api_ref/proc_independ.rst | 1 -
 docs/guides/api_ref/spp_mirror.rst    | 2 +-
 docs/guides/api_ref/spp_nfv.rst       | 2 +-
 docs/guides/api_ref/spp_primary.rst   | 8 +++++++-
 docs/guides/api_ref/spp_vf.rst        | 4 ++--
 5 files changed, 11 insertions(+), 6 deletions(-)

diff --git a/docs/guides/api_ref/proc_independ.rst b/docs/guides/api_ref/proc_independ.rst
index 2e73a1d..52e3002 100644
--- a/docs/guides/api_ref/proc_independ.rst
+++ b/docs/guides/api_ref/proc_independ.rst
@@ -62,5 +62,4 @@ Response example
         "type": "nfv",
         "client-id": 2
       }
-      ...
     ]
diff --git a/docs/guides/api_ref/spp_mirror.rst b/docs/guides/api_ref/spp_mirror.rst
index a19eac7..1d4efee 100644
--- a/docs/guides/api_ref/spp_mirror.rst
+++ b/docs/guides/api_ref/spp_mirror.rst
@@ -316,7 +316,7 @@ Detach tx port of ``ring:1`` from component named ``mr1``.
 .. code-block:: console
 
     $ curl -X PUT -H 'application/json' \
-      -d '{"action": "detach", "port": "ring:0", "dir": "tx"} \
+      -d '{"action": "detach", "port": "ring:0", "dir": "tx"}' \
       http://127.0.0.1:7777/v1/mirrors/1/components/mr1/ports
 
 
diff --git a/docs/guides/api_ref/spp_nfv.rst b/docs/guides/api_ref/spp_nfv.rst
index c30da06..949f772 100644
--- a/docs/guides/api_ref/spp_nfv.rst
+++ b/docs/guides/api_ref/spp_nfv.rst
@@ -85,7 +85,7 @@ Response example
       "client-id": 1,
       "status": "running",
       "ports": [
-        "phy:0", "phy:1", "vhost:0", "vhost:1", "ring:0", "ring:1", ...
+        "phy:0", "phy:1", "vhost:0", "vhost:1", "ring:0", "ring:1"
       ],
       "patches": [
         {
diff --git a/docs/guides/api_ref/spp_primary.rst b/docs/guides/api_ref/spp_primary.rst
index 3cbad10..276c7e9 100644
--- a/docs/guides/api_ref/spp_primary.rst
+++ b/docs/guides/api_ref/spp_primary.rst
@@ -125,7 +125,13 @@ Response example
           "tx": 0,
           "tx_drop": 0
         },
-        ...
+        {
+          "id": 2,
+          "rx": 0,
+          "rx_drop": 0,
+          "tx": 0,
+          "tx_drop": 0
+        }
       ]
     }
 
diff --git a/docs/guides/api_ref/spp_vf.rst b/docs/guides/api_ref/spp_vf.rst
index c406938..173adc4 100644
--- a/docs/guides/api_ref/spp_vf.rst
+++ b/docs/guides/api_ref/spp_vf.rst
@@ -139,7 +139,7 @@ Response example
     {
       "client-id": 1,
       "ports": [
-        "phy:0", "phy:1", "vhost:0", "vhost:1", "ring:0", "ring:1", ...
+        "phy:0", "phy:1", "vhost:0", "vhost:1", "ring:0", "ring:1"
       ],
       "components": [
         {
@@ -440,7 +440,7 @@ Request example
 .. code-block:: console
 
     $ curl -X PUT -H 'application/json' \
-      -d '{"action": "detach", "port": "vhost:0", "dir": "tx"} \
+      -d '{"action": "detach", "port": "vhost:0", "dir": "tx"}' \
       http://127.0.0.1:7777/v1/vfs/1/components/fwd1/ports
 
 Response
-- 
2.17.1

